signer/.air.toml

```diff
@@ -0,0 +1,37 @@
+root = "."
```

Comment:
What exactly is this file used for?

Reply:
its a [hot reloader for go](https://github.com/cosmtrek/air) i tend to use for go service development... i suppose don't have to commit this

Reply:
Going to leave this up to the team to decide :)